src/sentry/models/integration.py

```diff
@@ -87,3 +87,29 @@ def add_organization(self, organization_id, default_auth_id=None, config=None):
             return False
         else:
             return True
+
+    def add_project(self, project_id, config=None):
+        """
+        Add a project to this integration. Requires that a
+        OrganizationIntegration must exist before the project can be added.
+
+        Returns True if the ProjectIntegration was created
+        """
+        from sentry.models import Project
+        if not OrganizationIntegration.objects.filter(
+            organization_id=Project.objects.get(id=project_id).organization_id,
+            integration=self,
+        ).exists():
```

Comment:
kind of ugly tbh. how about using a variable like: 
organizationobjects = BigModel.bigquery()
if not organizationobjects.exist():